### DISCUSSION TITLE
Muffled Audio on HTML5

### DISCUSSION THREAD

**Discussion by TheHENOOB** [HaxeFlixel/flixel#2361] [Q&A]
I don't know if some people passed from this but my games has a problem with the game audio being muffled or strange to hear in Firefox (Maybe in other browsers too), I don't know if this was fixed because i found that this bug is old and i don't find any answer for it, any answers?

**Comment by RichardBray** [comment]
I don't think I've had this issue before. Maybe it's a bug in the newer version of flixel. 🤷‍♂️
Is your audio in .ogg format?

**Reply by TheHENOOB** [reply]
> I don't think I've had this issue before. Maybe it's a bug in the newer version of flixel. 🤷‍♂️
> Is your audio in .ogg format?

I make two file formats for a single audio file, i use .mp3 to HTML5 and .ogg to Desktop/CPP targets, I convert them on audacity

**Reply by MSGhero** [reply]
This happened to one of my songs in an OpenFL game but not the other ones. Not sure why, html5 target

**Reply by TheHENOOB** [reply]
> 
> 
> This happened to one of my songs in an OpenFL game but not the other ones. Not sure why, html5 target

Which song?

**Reply by MSGhero** [reply]
https://www.newgrounds.com/portal/view/792154, Endless Handbag - Carpet Mix @TheHENOOB

**Reply by RichardBray** [reply]
You can use .ogg for HTML5 as well. I'm not sure about the warnings, I don't think I've seen them before but I don't think they're related.

**Reply by TheHENOOB** [reply]
I made a video about this issue to make it more clear (Use a headset)

https://user-images.githubusercontent.com/74883505/127798972-98839e9d-bb90-406b-a0fc-2d993a12e45b.mp4

**Reply by RichardBray** [reply]
Will you be comfortable with temporarily open-sourcing your code so I can try it out on my machine? I'm using a mac so maybe it will build the sound different 🤷. Also, have you tried using .ogg for html5 as well?

**Reply by TheHENOOB** [reply]
> Will you be comfortable with temporarily open-sourcing your code so I can try it out on my machine? I'm using a mac so maybe it will build the sound different 🤷. Also, have you tried using .ogg for html5 as well?

I will make a audio demo instead of open source the game, i also tried to use .ogg it was the same result

**Reply by TheHENOOB** [reply]
I released the game and the source code, if you want to check it you can go there

https://github.com/TheHENOOB/CarTrip
